ansible/roles/pmix/tasks/main.yaml

```diff
@@ -12,17 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
----
-
 - name: Include OS Family Dependent Vars
   ansible.builtin.include_vars: '{{ item }}'
   with_first_found:
-  - '{{ ansible_distribution|lower }}-{{ ansible_distribution_version }}.yml'
-  - '{{ ansible_distribution|lower }}-{{ ansible_distribution_major_version }}.yml'
-  - '{{ ansible_distribution|lower }}.yml'
-  - '{{ ansible_os_family|lower }}-{{ ansible_distribution_version }}.yml'
-  - '{{ ansible_os_family|lower }}-{{ ansible_distribution_major_version }}.yml'
-  - '{{ ansible_os_family|lower }}.yml'
+  - files:
```

Comment:
Was this change tested?  I'm not super familiar with the syntax, but a quick lookup shows it being something like:

```yaml
- name: Load a variable file based on the OS type, or a default if not found. Using free-form to specify the file.
  ansible.builtin.include_vars: "{{ lookup('ansible.builtin.first_found', params) }}"
  vars:
    params:
      files:
        - '{{ansible_distribution}}.yaml'
        - '{{ansible_os_family}}.yaml'
        - default.yaml
      paths:
        - 'vars'
 ```

Reply:
We resolved this by eliminating the change from the v6 branch in #64